src/pushInfo/PushInfo.tsx

```diff
@@ -11,17 +16,22 @@ export type PushInfoProps = Readonly<{
   onAnimationEnd?: Function
 }>
 
-export class PushInfo extends Component<PushInfoProps> {
-  componentDidMount() {
-    if (this.props.onAnimationEnd) {
-      setTimeout(this.props.onAnimationEnd, animationDuration + animationDelay)
+export const PushInfo = ({
+  className,
+  icon,
+  headline,
+  content,
+  onAnimationEnd,
+}: PushInfoProps): JSX.Element => {
+  useEffect(() => {
+    if (onAnimationEnd) {
+      setTimeout(onAnimationEnd, animationDuration + animationDelay)
     }
-  }
+  }, [onAnimationEnd])
 
-  render() {
-    const { className, icon, headline, content } = this.props
-    return (
-      <StyledPushInfo className={cc(['kirk-pushInfo', className])}>
+  return (
+    <StyledPushInfoWrapper className={cc(['kirk-pushInfo', className])}>
+      <StyledPushInfo>
         {icon && <div className="kirk-pushInfo-icon">{icon}</div>}
         <div>
           <h2
```

Comment:
Can we transform that on styled-components using a typographic component as base?

Reply:
We should, but not really the scope of this PR.